backend/core/domains/payments/services/gateway_monitoring_service.py

```
@@ -0,0 +1,606 @@
+            consecutive_failures = cls._get_consecutive_failures(gateway_code)
+            last_success = cls._get_last_success(gateway_code)
+
+            return GatewayHealthCheck(
+                gateway_code=gateway_code,
+                is_healthy=is_healthy,
+                response_time_ms=response_time_ms,
+                consecutive_failures=consecutive_failures,
+                last_success=last_success
+            )
+
+        except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)